.github/workflows/run-model-tests-1.yml

```diff
@@ -59,4 +59,5 @@ jobs:
 
       - name: run test
         run: |
-          TEST_SUBFOLDER=models docker compose run runtests
+          shopt -s nullglob
+          TEST_SUBFOLDER=$(ls tests/models/test_{a..l}*.py) docker compose run runtests
```

Comment:
I'm guessing this split is because the model tests have been slow?

I think the reason is the NOIRLab server; I've been seeing hangs trying to get images from there.  One thing we might want to do is separate out the tests of pulling exposures from that server, and have them run only when an env var is set; then, by default, disable that env var, so the github tests won't hang up when the NOIRLab server is slow.  The exposure(s) we need for the bulk of the decam tests can go to one of our archives where we store our other test images and such.

Reply:
I don't think I'm downloading anything from NOIRLab anymore. The tests are slow because there are a lot of them and many of them produce an entire datastore for the test. I might look into shortening that.